js/android/app/src/main/AndroidManifest.xml

```diff
@@ -2,9 +2,10 @@
   package="tech.berty.app">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.VIBRATE"/>
     <uses-permission android:name="android.permission.CAMERA" />
+
     <application
-      android:networkSecurityConfig="@xml/network_security_config"
```

Comment:
You should not remove this or the webui won't work anymore.
I think the same problem will occur for the tcp_reverse w/ metro on localhost.

Reply:
Metro access is in a very bad shape on master, on this branch it's automatic, doesn't even need the reverse I think